docs/getting-started.md

```diff
@@ -1 +1,137 @@
-This is a placeholder for some docs
+## 1. Overview
+Kubeflow is a novel open source tool for Machine Learning workflow orchestration on Kubernetes.
+It has great powers, but deploying it may not be so easy, depending on how and where you deploy
+your Kubernetes. This tutorial will show you an easy way to deploy Kubeflow using MicroK8s, a
+lightweight version of Kubernetes, in a few simple steps.
+
+### What you'll learn
+
+* How to deploy MicroK8s on Ubuntu
+* How to deploy Kubeflow using MicroK8s
+* How to access your Kubeflow dashboard
+
+### What you'll need
+* Desktop or Virtual Machine with Ubuntu 16.04 or above
+* A minimum of 4 CPU, 14GB RAM, 50GB Disk (recommended 8 CPU, 16GB RAM, 60GB Disk)
+* Hyper V, if using Multipass on a Windows machine (not available on Windows 10 Home)
```

Comment:
Do you actually need Hyper-V. Or do you just need a VM? Because the Windows installer works with VirtualBox.

Reply:
That's a good question for @tvansteenburgh

Reply:
> That's a good question for @tvansteenburgh

Yeah, VBox is fine too. In general we should just point folks to the installation reqs/instructions for MicroK8s so we don't have to constantly keep these in sync.